databricks_cli/pipelines/cli.py

```diff
@@ -208,17 +213,45 @@ def reset_cli(api_client, pipeline_id):
 @provide_api_client
 def run_cli(api_client, pipeline_id):
     """
+    (Deprecated) "run" command is deprecated, please use "start" command instead.
+
     Starts the execution of a delta pipelines run by starting the cluster and processing data.
 
     Usage:
 
     databricks pipelines run --pipeline-id 1234
     """
+    click.echo("DeprecationWarning: \"run\" command is deprecated," +
+               " please use \"start\" command instead")
     _validate_pipeline_id(pipeline_id)
-    PipelinesApi(api_client).run(pipeline_id)
+    PipelinesApi(api_client).start(pipeline_id, full_refresh=False)
     click.echo("Run triggered for pipeline {}".format(pipeline_id))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Starts the execution of a delta pipeline run')
+@click.option('--pipeline-id', default=None, type=PipelineIdClickType(),
+              help=PipelineIdClickType.help)
+@click.option('--full-refresh', default=False, type=bool,
+              help='If true, truncates tables and creates new checkpoint' +
+                   ' folders so data is reprocessed from scratch')
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def start_cli(api_client, pipeline_id, full_refresh):
+    """
+    Starts the execution of a delta pipelines run by starting the cluster and processing data.
+
+    Usage:
+
+    databricks pipelines start --pipeline-id 1234 --full-refresh=(false|true)
+    """
+    _validate_pipeline_id(pipeline_id)
+    PipelinesApi(api_client).start(pipeline_id, full_refresh=full_refresh)
```

Comment:
Can this fail? If so, how do the error messages look?

Reply:
If the interface itself is not reachable, error looks something like this

`Error: b'<html>\n<head>\n<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>\n<title>Error 403 Invalid access token.</title>\n</head>\n<body><h2>HTTP ERROR 403</h2>\n<p>Problem accessing /api/2.0/pipelines/{pipeline_id}/start. Reason:\n<pre>    Invalid access token.</pre></p>\n</body>\n</html>\n'
`
Error from backend looks like this:
```
Error: RESOURCE_DOES_NOT_EXIST
The specified pipeline {pipeline_id} was not found.

```

I believe pattern is Error: <error thrown either from backend or the library>